### PR TITLE
OSSM-5038: Enable kind internal registry

### PR DESCRIPTION
**Pull request by jwendell**
This avoids pushing a temporary image to quay within the kind integration tests.

### CODE REVIEW

File: tests/integration/integ-suite-kind.sh [hunk 1]
```diff
@@ -16,42 +16,50 @@
...
-
-function cleanup_kind_cluster() {
-  echo "Test exited with exit code $?."
-  echo "Cleaning up kind cluster"
-  kind delete cluster --name istio-operator || true
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT=$(dirname "$(dirname "${SCRIPTPATH}")")
+
+# shellcheck source=common/scripts/kind_provisioner.sh
+source "${ROOT}/common/scripts/kind_provisioner.sh"
```
Comment (jwendell) [inline tests/integration/integ-suite-kind.sh:23]: This imports the function `setup_kind_registry()` meaning we no longer have to implement it on our own (see removed lines above)

**Review by luksa — APPROVED** [review]

**Pull request merged**